packages/framework/src/Component/Router/FriendlyUrl/FriendlyUrlFacade.php

```diff
@@ -108,7 +112,6 @@ protected function resolveUniquenessOfFriendlyUrlAndFlush(FriendlyUrl $friendlyU
         }
 
         $this->em->persist($friendlyUrl);
-        $this->em->flush();
```

Comment:
the method should not be called `...AndFlush` anymore

Reply:
changed